.scrutinizer.yml

```diff
@@ -0,0 +1,35 @@
+build:
+    nodes:
+        analysis:
+            environment:
+                php:
+                    version: 7.2
+                node:
+                    version: 6.14
```

Comment:
Do we need to use the latest LTS of 10 or doesn't it matter? Version 6 is currently maintenance only.

Reply:
I've updated the version to the current LTS, 6.14 just happened to be the version I have installed locally. Any version that Webpack supports (currently >=6.11) should be sufficient.